src/data/setoid/basic.lean

```diff
@@ -359,4 +359,17 @@ def correspondence (r : setoid α) : {s // r ≤ s} ≃o setoid (quotient r) :=
       t.1.trans' (t.1.symm' $ t.2 $ eq_rel.1 hx) $ t.1.trans' ht $ t.2 $ eq_rel.1 hy,
       λ h x y hs, let ⟨a, b, hx, hy, Hs⟩ := hs in ⟨a, b, hx, hy, h Hs⟩⟩ }
 
+@[simp]
+lemma setoid.rel_top  : (⊤ : setoid α).rel = ⊤ := rfl
```

Comment:
This belongs right below `complete_lattice` on line 139. Can you add a similar lemma for `bot` too?

Also looking above, I guess it should be called `top_def` to match `inf_def`.

Reply:
Done